_includes/head.html

```diff
@@ -55,9 +55,9 @@
     <![endif]-->
 
     <!-- Custom stylesheet -->
-    <link rel="stylesheet" type="text/css" href="/css/main.css" />
+    <link rel="stylesheet" type="text/css" href="/css/main.css">
 
     <!-- metadata -->
-    <meta name="og:title" content="{{ page.title }}"/>
-    <meta name="og:image" content="/images/bazel-og-image.png"/>
+    <meta name="og:title" content="{{ page.title }}">
+    <meta name="og:image" content="https://www.bazel.build/images/bazel-og-image.png">
```

Comment:
Added a separate `assets_url` variable since `main_site_url` should be empty string in order to make testing a local instance of the site manageable (by using relative paths). I was not able to find a way to have variables with different values depending on prod vs dev environments though.